### ISSUE TITLE
PS3 freezes upon loading ISO over the network when fan control != SYSCON

### ISSUE THREAD

**Issue by GottemHams** [aldostools/webMAN-MOD#466]
I was reading about webMAN's fan control options and apparently `SYSCON` is simply the factory setting which generally speaking is not very good. So I tried changing it to `Auto` and rebooted the PS3 like webMAN asks me to. After rebooting I tried to mount a PS3ISO through ps3netsrv but this causes the XMB to immediately hang, I can't even get out of the `webMAN Games` submenu since none of the controller buttons work. I have to hold the physical power button for about 10 seconds to force it off.

After the reboot I changed it back to `SYSCON` and the exact same ISO now mounts without issues and I can actually go and play the game. Then I tried one more time to set it to `Auto` and the same problem occurs.

Versions: HFW 4.87, PS3HEN 3.02

**Comment by aldostools** [comment]
PS3HEN has a known issue with dynamic fan control. Try manual fan mode.

In simple terms, when you launch a game or homebrew the power consumption increases generating more heat, the dynamic fan changes the fan policy in memory to correct the temperature. LV1 (hypervisor) detects the change in memory and enters in panic mode (freeze the console).

SYSCON mode operates with a fan speed between 20% and 27% and only increases the speed when the temperature is over 80C.

![image](https://user-images.githubusercontent.com/974897/104626778-a931c500-566c-11eb-8312-d659786f756b.png)

If you set the fan speed above 27% (try 35% or 40%) the fan will be always much better than SYSCON and you won't suffer of the freeze produced by the hypervisor.

**Comment by GottemHams** [comment]
I see, so dynamic fan control is pretty much impossible without CFW, since I guess that's the only way to prevent the hypervisor from panicking?

As for setting it to manual, I assume that if SYSCON would decide to go to e.g. 40% while manual is set to 35%, it would still allow for it to run at that 40%? Like, manual is just the base level and anything faster is allowed. My room gets quite hot over the summer. :>

Just curious: how does manual mode achieve the fan change without causing a panic?

**Comment by aldostools** [comment]
Manual mode overrides SYSCON mode. 

If you see the graphic in my previous post, SYSCON only would go above 35% if the temperature is over 85C which is already too hot. If you reach that temperature you will get a warning and must shutdown the console as soon as possible.

If your console is getting above 80C often means that your fan settings are too low.

**Comment by Dee-san** [comment]
> I see, so dynamic fan control is pretty much impossible without CFW, since I guess that's the only way to prevent the hypervisor from panicking?

Seems so - my friend with PS3 Slim (CFW compatible) never had these issues...

> Just curious: how does manual mode achieve the fan change without causing a panic?

Manual mode doesn't change the fan speed dynamically, so it also doesn't access the memory and cannot trigger a LV1 panic (right?)

Just wished there was CFW on late PS3 Slims... but at least I can take backups and run homebrew :)

**Comment by aldostools** [comment]
Right, manual doesn't change memory at random periods, thus the LV1 panic doesn't happen.

CFW doesn't have this issue because LV1 is patched. But HEN cannot patch LV1.

**Comment by GottemHams** [comment]
With SYSCON after having played a game for a few hours:
CPU: 63C
RSX: 66C
Fan: 28%

I set the fan to 30% for now because 35% is a bit too loud imo. Temps dropped almost immediately, these are from a few minutes after closing the game: CPU 59C and RSX 60C. Resumed play for another few hours and now, after exiting to XMB it's 58C/59C.

Of course this is all just side information, the actual issue from my OP had been fixed after moving away from dynamic mode. :> One little remaining question though: does the PS2 emulator generate much more heat? I noticed the default in webMAN is 40%, but that's pretty damn loud. :D I can't check this myself because I haven't gotten the emulator working yet.

**Comment by Joonie86** [comment]
No further feedback
